WalletWasabi.Fluent/Helpers/TransactionFeeHelper.cs

```diff
@@ -1,12 +1,15 @@
 using System.Collections.Generic;
 using System.Linq;
 using NBitcoin;
+using WalletWasabi.Fluent.ViewModels.Wallets.Send;
 using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.Helpers;
 
 public static class TransactionFeeHelper
 {
+	public const decimal FeePercentageThreshold = 130;
```

Comment:
Just to be similar to https://github.com/zkSNACKs/WalletWasabi/blob/5e4f6f791f033b23c7c1ad31d41a20f873bbc84a/WalletWasabi/Blockchain/TransactionBuilding/ChangelessTransactionCoinSelector.cs#L15

```suggestion
	public const decimal FeePercentageThreshold = 125;
```